### PR TITLE
added Orders feature

### PR DESCRIPTION
**Pull request by ivansamoyloff**
#9

### CODE REVIEW

File: src/components/Orders/Orders.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  height: 180px;
+  padding: 20px;
+  color: white;
+}
+
+.Orders-intro {
+  font-size: large;
+}
+
+table {
```
Comment (dantix) [inline src/components/Orders/Orders.css]: All code should be properly indented, we use 2 spaces to indent, use `editorconfig` plugin for consistent indentation in the project.

Comment (dantix) [inline src/components/Orders/Orders.css]: Use classes instead of tag names

File: src/components/Orders/Orders.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+th, td {
+	width: 10%;
+}
+.Active{
+	color: green;
+}
+.NonActive{
+	color: red;
+}
+.orderid{
```
Comment (dantix) [inline src/components/Orders/Orders.css]: Use consistent naming case

File: src/components/Orders/Orders.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import logo from '../../logo.svg';
+import './Orders.css'
+import {order1} from './orderslist.js'
+import {order2} from './orderslist.js'
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Importing multiple variables from one modules can be done in one statement

File: src/components/Orders/Orders.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+import {order3} from './orderslist.js'
+import {order4} from './orderslist.js'
+
+var array = [order1, order2, order3, order4];
+
+export default class Orders extends React.Component {
+	constructor(props){
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Code is not properly indented

File: src/components/Orders/Orders.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	
+  render() {
+    return (
+    	<div className="Orders">
+	        <div className="Orders-header">
+	          <img src={logo} className="Orders" alt="logo"/>
+	          <h2>Welcome to Sloboda taxi</h2>
+	        </div>
+	        <div>
+        	<table className="table table-striped table-hover">
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Order list should be a separate component that will get orders via props and render them

File: src/config/RouterConfig.js [hunk 6]
```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import {Router, Route, IndexRoute, browserHistory} from 'react-router';
 import OrderPage from '../components/OrderPage/OrderPage';
-import SignUp from '../components/SignUpPage/SignUpPage';
+import SignUpForm from '../components/SignUpForm/SignUpForm';
```
Comment (dantix) [inline src/config/RouterConfig.js]: These changes are not relevant to the order list

File: public/index.html [hunk 7]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="dist/spectre.min.css" />
```
Comment (dantix) [inline public/index.html]: This is not necessary, spectre is already included in `App.js`

File: src/components/Orders/Orders.css (hunk 1 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.css]: for th and td tags also use classes?

File: src/components/Orders/Orders.css (hunk 2 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.css]: fixed

File: src/components/Orders/Orders.js (hunk 3 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.css (hunk 1 above)
Comment (dantix) [inline src/components/Orders/Orders.css]: yes

File: src/components/Orders/Orders.js [hunk 8]
```diff
@@ -1,74 +1,75 @@
...
-	        			<td>{param["driver_id"]}</td>
-	        			<td>{param["from"]}</td>
-	        			<td>{param["to"]}</td>
-	        			<td>{param["price"]}</td>
-	        			<td>{param["comment"]}</td>
-	        			<td>{param["created_at"]}</td>
-	        			<td>{param["updated_at"]}</td>
-	        		</tr>
-			);
+order(param) {
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Wrong indentation

File: src/components/Orders/Orders.js [hunk 9]
```diff
@@ -1,74 +1,75 @@
...
+            <td className="Table-cell">{param["from"]}</td>
+            <td className="Table-cell">{param["to"]}</td>
+            <td className="Table-cell">{param["price"]}</td>
+            <td className="Table-cell">{param["comment"]}</td>
+            <td className="Table-cell">{param["created_at"]}</td>
+            <td className="Table-cell">{param["updated_at"]}</td>
+           </tr>
+        );
+};
+orderlist(){
```
Comment (dantix) [inline src/components/Orders/Orders.js]: It is not a separate component

File: src/components/Orders/Orders.js [hunk 10]
```diff
@@ -1,74 +1,75 @@
...
-		super(props);	
-		this.ifactive = this.ifactive.bind(this);
-		this.order = this.order.bind(this);
-	}
+  constructor(props){
+    super(props);
+    this.ifactive = this.ifactive.bind(this);
+    this.order = this.order.bind(this);
+  }
 ifactive(parameters) {
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Wrong indentation

File: src/components/Orders/Orders.css [hunk 11]
```diff
@@ -17,22 +17,22 @@
   font-size: large;
 }
 
-table {
-	margin: 0 auto;
-	border-collapse: collapse;
+.table {
```
Comment (dantix) [inline src/components/Orders/Orders.css]: Are you sure that you need this styles?

File: src/components/Orders/Orders.js (hunk 10 above)
Comment (dantix) [inline src/components/Orders/Orders.js]: `parameters` is too generic name for variable

File: src/components/Orders/Orders.js [hunk 12]
```diff
@@ -1,74 +1,75 @@
...
-	        	</thead>
-	        	<tbody>
-	        		{array.map(this.order)}
-	        	</tbody>		
-        	</table>
-        	</div>
-      </div>  
+render() {
+  return (
+          <div className="Orders">
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Indentation

File: src/components/Orders/Orders.css (hunk 11 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.css]: fixed

File: src/components/Orders/Orders.js (hunk 10 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.js (hunk 8 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.js (hunk 9 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.js (hunk 12 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.js [hunk 13]
```diff
@@ -1,75 +1,18 @@
...
-          <div className="Orders-header">
-            <img src={logo} className="Orders" alt="logo"/>
-            <h2>Welcome to Sloboda taxi</h2>
-          </div>
-          {this.orderlist()}
-      </div>
+  render() {
+    return (
+            <div className="Orders">
+            <div className="Orders-header">
```
Comment (dantix) [inline src/components/Orders/Orders.js]: Wrong indentation

File: src/components/Orders/Orders.js [hunk 14]
```diff
@@ -1,75 +1,18 @@
...
+              <img src={logo} className="Orders" alt="logo"/>
+              <h2>Welcome to Sloboda taxi</h2>
+            </div>
+            <Orderslist />
```
Comment (dantix) [inline src/components/Orders/Orders.js]: You should pass orders as prop to `Orderslist` component. Use camel case for Component naming.

File: src/components/Orders/Orderslist.js [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+              <td className="Table-cell">{param["price"]}</td>
+              <td className="Table-cell">{param["comment"]}</td>
+              <td className="Table-cell">{param["created_at"]}</td>
+              <td className="Table-cell">{param["updated_at"]}</td>
+            </tr>
+          );
+  };
+  orderlist() {
+    return(
+            <div className="Order-list">
```
Comment (dantix) [inline src/components/Orders/Orderslist.js]: Wrong indentation

File: src/components/Orders/Orderslist.js [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+                </tr>
+              </thead>
+              <tbody>
+                {array.map(this.order)}
+              </tbody>
+              </table>
+            </div>
+          );
+  };
+  render() {
```
Comment (dantix) [inline src/components/Orders/Orderslist.js]: Wrong indentation

File: src/components/Orders/Orders.js (hunk 13 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orders.js (hunk 14 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/Orderslist.js (hunk 15 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orderslist.js]: fixed

File: src/components/Orders/Orderslist.js (hunk 16 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orderslist.js]: fixed

File: src/components/Orders/Orders.css [hunk 17]
```diff
@@ -0,0 +1,33 @@
+.Orders {
+  text-align: center;
+}
+
+.Orders {
```
Comment (EvgeniiK) [inline src/components/Orders/Orders.css]: You already have it in the first row, please combine it.

File: src/components/Orders/Orders.js [hunk 18]
```diff
@@ -11,7 +13,7 @@ export default class Orders extends React.Component {
         <img src={logo} className="Orders" alt="logo"/>
         <h2>Welcome to Sloboda taxi</h2>
       </div>
-      <OrdersList />
+      <OrdersList source={[order1,order2,order3,order4]} />
```
Comment (dantix) [inline src/components/Orders/Orders.js]: `orders` is better name for such variable

File: src/components/Orders/OrdersList.js [hunk 19]
```diff
@@ -34,7 +31,7 @@ export default class OrdersList extends React.Component {
       </tr>
     );
   };
-  orderlist() {
+  orderlist(orderdata) {
```
Comment (dantix) [inline src/components/Orders/OrdersList.js]: This function is not necessary and can be inlined in `render` method

File: src/components/Orders/OrdersList.js [hunk 20]
```diff
@@ -20,8 +17,8 @@ export default class OrdersList extends React.Component {
   };
   order(param) {
     return(
-      <tr>
-        <td id={param["id"]} className="Table-cell Order-id">{param["id"]}</td>
+      <tr id={param["id"]} key={param["id"]}>
```
Comment (dantix) [inline src/components/Orders/OrdersList.js]: Use dot-notation to access object properties when it is possible

File: src/components/Orders/Orders.css (hunk 17 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.css]: these classes for different elements, I'm just rename class, ok?

File: src/components/Orders/OrdersList.js [hunk 21]
```diff
@@ -20,8 +17,8 @@ export default class OrdersList extends React.Component {
   };
   order(param) {
```
Comment (dantix) [inline src/components/Orders/OrdersList.js]: `param` is bad name for variable, `order` (in noun form) - bad name for a function

File: src/components/Orders/Orders.css (hunk 17 above)
Comment (dantix) [inline src/components/Orders/Orders.css]: If there are 2 styles with same selector their contents are combined, so you can not use same class name to define styles for different things

Comment (ivansamoyloff) [inline src/components/Orders/Orders.css]: these not mine changes, this was when I pulled the project from git

File: src/components/Orders/Orders.js (hunk 18 above)
Comment (ivansamoyloff) [inline src/components/Orders/Orders.js]: fixed

File: src/components/Orders/OrdersList.js (hunk 21 above)
Comment (ivansamoyloff) [inline src/components/Orders/OrdersList.js]: fixed

File: src/components/Orders/OrdersList.js (hunk 20 above)
Comment (ivansamoyloff) [inline src/components/Orders/OrdersList.js]: fixed

File: src/components/Orders/OrdersList.js (hunk 19 above)
Comment (ivansamoyloff) [inline src/components/Orders/OrdersList.js]: fixed

File: src/components/Orders/OrdersList.js [hunk 22]
```diff
@@ -15,23 +14,23 @@ export default class OrdersList extends React.Component {
       return(<td className="Table-cell Non-active">non-active</td>);
     }
   };
-  order(param) {
+  displayorder(orderdata) {
```
Comment (ivansamoyloff) [inline src/components/Orders/OrdersList.js]: fixed

**Pull request merged**